src/test/ui-fulldeps/auxiliary/issue-18502.rs

```diff
@@ -12,7 +12,7 @@ impl Show for Foo {
     fn fmt(&self) {}
 }
 
-fn bar<T>(f: extern "Rust" fn(&T), t: &T) { }
+fn bar<T>(f: fn(&T), t: &T) {}
```

Comment:
Oh, hm. My eyes glanced over that initially. Removing code seems more of an actual problem even for fulldeps. I wasn't aware rustfmt did that. I'm not sure if that matters here but it's got me concerned enough to worry about doing this at all. And the issue (#18502) is from pre-1.0 Rust which makes it awkward for me to tell if this matters.

I'm fairly confident that the other tests aren't sensitive to formatting (or at least they aren't intending to be) but this is something different.